pdns/dnsdist-lua-bindings.cc

```diff
@@ -105,7 +105,7 @@ void setupLuaBindings(bool client)
       g_pools.setState(localPools);
       s->pools.erase(pool);
     });
-  g_lua.registerFunction<void(DownstreamState::*)()>("getOutstanding", [](const DownstreamState& s) { g_outputBuffer=std::to_string(s.outstanding.load()); });
+  g_lua.registerFunction<int(DownstreamState::*)()>("getOutstanding", [](const DownstreamState& s) { return s.outstanding.load(); });
```

Comment:
If you don't mind, please make the return value the same type than the `outstanding` member (`uint64_t`) otherwise we'll likely have conversion issues down the road.